index.html

```diff
@@ -7,40 +7,42 @@
   <link href="styles/base.css" type="text/css" rel="stylesheet" />
   <link href="styles/layout.css" type="text/css" rel="stylesheet" />
   <link href="styles/module.css" type="text/css" rel="stylesheet" />
-
+  <script id="article-template" type="text/x-handlebars-template"
```

Comment:
this is your problem
gotta close your script tag,
this is throwing off the dom structure and messing with you jquery import